app/Main.hs

```diff
@@ -73,12 +74,29 @@ cliParser =
               <*> pure gitInfo
           )
  where
+  render :: Parser CLI
+  render = RenderMap <$> strArgument (metavar "FILE") <*> subOpts
```

Comment:
Maybe the `metavar` should be something more like `SCENARIO` instead of just `FILE`.  In fact, just as with the `--scenario` flag, you do not have to give it a filename, but it actually works to write e.g. `swarm map Fun/horton` rather than `swarm map data/scenarios/Fun/horton.yaml`.